language-service/src/parser/jsonParser.ts

```diff
@@ -764,11 +764,31 @@ export class ObjectASTNode extends ASTNode {
 
 			if (schema.properties) {
 				const propSchema: JSONSchema = schema.properties[propertyKey];
-				const ignoreKeyCase: boolean = ASTNode.getIgnoreKeyCase(propSchema);
 
-				if (ignoreKeyCase) {
-					const matchedKeys: ASTNodeMap = findMatchingProperties(propertyKey);
-					return Object.keys(matchedKeys).length > 0;
+				if (propSchema) {
```

Comment:
Suggestion. It seems to me that this newly added code is almost a duplication of previous hasProperty function. You can probably do something like this
```
const hasAliasOrProperty(key:string) => hasProperty(key, schema.properties) || hasProperty(key, schema.properties[key].aliases)
```